site/gatsby-site/src/components/checklists/ChecklistsIndex.js

```diff
@@ -0,0 +1,119 @@
+      <div className="flex flex-col gap-4 bg-gray-100 border-1 border-gray-200 rounded shadow-inner p-4">
+        {checklists.map((checklist) => (
+          <Card className="bg-white shadow" key={checklist.id}>
+            <div className="flex items-center gap-2 flex-wrap border-b border-gray-200 p-4">
+              <LocalizedLink className="mr-auto" to={`/apps/checklists?id=${checklist.id}`}>
+                <h2 className="mb-0">{checklist.name}</h2>
+              </LocalizedLink>
+              <Button
+                color="light"
+                onClick={async () => {
+                  const newChecklist = {
+                    ...checklist,
+                    _id: undefined,
+                    id: generateID(),
+                    name: checklist.name + t(' (Clone)'),
+                  };
+
+                  try {
+                    await insertChecklist({
+                      variables: { checklist: removeTypename(newChecklist) },
+                    });
+                    setChecklists((checklists) => {
+                      const newChecklists = [...checklists];
+
+                      newChecklists.splice(checklists.indexOf(checklist), 0, newChecklist);
+                      return newChecklists;
+                    });
+                  } catch (e) {
+                    console.log(e);
+                  }
+                }}
+              >
+                <Trans>Clone</Trans>
+              </Button>
+              <Button color="light" onClick={() => alert('Coming soon')}>
+                <Trans>Subscribe</Trans>
+              </Button>
+              <DeleteButton
+                type="button"
+                onClick={async () => {
+                  try {
+                    await deleteChecklist({ variables: { query: { id: checklist.id } } });
+                    setChecklists((checklists) => checklists.filter((c) => c.id != checklist.id));
+                  } catch (e) {
+                    console.log(e);
+                  }
+                }}
+              >
+                <Trans>Delete</Trans>
+              </DeleteButton>
+              <ExportDropdown {...{ checklist }} />
+            </div>
+            <ul className="flex gap-2 flex-wrap p-4">
+              {checklist.risks
+                .filter((r) => r.risk_status != 'Not Applicable')
+                .map((risk) => (
+                  <li key={risk.id} className="flex items-center gap-1 text-gray-600 mx-1">
+                    <FontAwesomeIcon
+                      icon={statusIcon(risk.risk_status)}
+                      className={`text-${statusColor(risk.risk_status)}-600`}
+                      title={risk.risk_status}
+                    />
+                    {risk.title}
+                  </li>
+                ))}
+            </ul>
+          </Card>
+        ))}
+      </div>
+    </>
+  );
+}
+
+const generateID = () => [0, 0, 0, 0].map(() => Math.random().toString(36).slice(-10)).join('');
```

Comment:
Seeing this on the new pass. Another spot for `uuidv4`?

Dare I say -- we should track ID generation in an issue somewhere. This can get lost easily and it sounds like its still a v1 question.